CHANGELOG.md

```diff
@@ -8,6 +8,9 @@
   device, ``BasicSimulator``, as the backend.
   [(#493)](https://github.com/PennyLaneAI/pennylane-qiskit/pull/493)
 
+* Backwards compatibility with Qiskit BackendV1 has now been implemented. Previously, only backends of type BackendV2 were supported but now users can choose to use BackendV1 as well.
```

Comment:
I believe it's the other way around - we previously only supported V1 backends. Adding the `backend.num_qubits` syntax supports V2.

Reply:
Yep. Utkarsh mentioned the same thing as well. Thanks Lillian!